mlflow/models/evaluation/base.py

```diff
@@ -667,21 +667,41 @@ def evaluate(
 
     :param model: A pyfunc model instance, or a URI referring to such a model.
 
+    :param data: One of the following:
+     - A numpy array or list of evaluation features, excluding labels.
+     - A Pandas DataFrame, or a spark DataFrame,
+       containing evaluation features and labels. If `feature_names` argument not specified,
+       all columns will be regarded as feature columns, otherwise column names which match
+       `feature_names` will be regarded as feature columns.
+
     :param model_type: A string describing the model type. The default evaluator
                        supports "regressor" and "classifier" as model types.
-    :param dataset: An instance of :py:class:`mlflow.models.evaluation.EvaluationDataset`
-                    containing features labels (optional) for model evaluation.
-    :param run_id: The ID of the MLflow Run to which to log results. If
-                   unspecified, behavior depends on the specified `evaluator`.
-                   When `run_id` is unspecified, the default evaluator logs
-                   results to the current active run, creating a new active run if
-                   one does not exist.
+
+    :param targets: (Optional) A numpy array or list of evaluation labels, if `data` is also a
+      numpy array or list.
+
+    :param label_col: (Optional) The string name of a column from `data` that contains
+      evaluation labels, if `data` is a DataFrame.
+
+    :param dataset_name: (Optional) The name of the dataset, must not contain double quotes (“).
+.
+    :param dataset_path: (Optional) the path to a serialized DataFrame
+      (e.g. a delta table, parquet file), must not contain double quotes (“).
+
+    :param feature_names: (Optional) If `data` argument is a feature data numpy array or list,
+      `feature_names` argument is a list of the feature names for each feature. If None, then
+      the `feature_names` will be generated using the format "feature_{feature_index}".
+      if `data` argument is a pandas dataframe or a spark dataframe, `feature_names` argument
+      is a list of the column names of the feature columns in the dataframe. If None, then
+      all columns except the label column will be regarded as feature columns.
+
     :param evaluators: The name of the evaluator to use for model evaluations, or
                        a list of evaluator names. If unspecified, all evaluators
                        capable of evaluating the specified model on the specified
                        dataset are used. The default evaluator can be referred to
                        by the name 'default'. If this argument is unspecified, then
-                       fetch all evaluators from the registry.
+                       fetch all evaluators from the registry. To get all available
+                       evaluators, call :py:func:`mlflow.models.evaluation.list_evaluators`
```

Comment:
Can we import `list_evaluators`, `EvaluationResult`, and `EvaluationArtifact` from `mlflow.models` (https://github.com/mlflow/mlflow/blob/master/mlflow/models/__init__.py#L33) and reference this as:

```suggestion
                       evaluators, call :py:func:`mlflow.models.list_evaluators()`
```

?